src/main/java/com/iri/training/model/UserComment.java

```diff
@@ -41,22 +39,14 @@ public int getUserID() {
 		public void setUserID(final int userID) {
 		this.userID = userID;
 	}
-		@Override
-		public String toString() {
-			final StringBuilder builder = new StringBuilder();
-			builder.append("description=");
-			builder.append(description);
-			builder.append("commentID=");
-			builder.append(commentID);
-			builder.append("commentDate=");
-			builder.append(commentDate);
-			builder.append("userID=");
-			builder.append(userID);
-
-
```

Comment:
It is considered best practice to use the string builder append method, than string concatenation.

In older versions of Java, the two approaches had significant performance differences. In current version the performance differences are for most cases non-existent. I would suggest to change all toString methods to use the string builder append method.